package.json

```diff
@@ -43,7 +43,7 @@
     "mutant-pull-reduce": "^1.1.0",
     "obv": "0.0.1",
     "patch-settings": "~1.1.0",
-    "patchcore": "^1.23.6",
+    "patchcore": "github:ssbc/patchcore#blogs",
```

Comment:
I assume a new `patchcore` would be published and this would be modified before merging

Reply:
yep!